src/idpconfgen/components/plots/plotfuncs.py

```diff
@@ -216,3 +216,67 @@ def plot_fracSS(
     plt.close("all")
 
     return errmsg
+
+def plot_bend_angles(
+        angles,
+        names,
+        *,
+        title=None,
+        xlabel="Backbone angles",
+        colors=['#D55E00', '#0072B2', 'k', 'g'],
+        ylabel="Bend Angle (Radians)",
+        title_fs=20,
+        xlabel_fs=15,
+        ylabel_fs=15,
+        xticks=None,
+        yticks=None,
+        xticks_labels=None,
+        yticks_labels=None,
+        increment=None,
+        xticks_fs=15,
+        yticks_fs=15,
+        fig_size=(10, 6),
+        filename='plot_bgeo.png',
+        dpi=300,
+        ):
+    """
+    Plot all torsion angle distributions as a scatter plot.
+    Defaults to Phi torsion angles.
+
+    Parameters
+    ----------
+    names : string
+        Names of the bend types
+    angles : np.ndarray, shape=(names, n_trimers)
+        Container of the Y axis data.
+    filename : str, optional
+        The file name with which the plot figure will be saved
+        in disk. Defaults to plot_torsions.png
+        You can change the file type by specifying its extention in
+        the file name.
+
+    Returns
+    -------
+    errmsg : list
+        List of errors to print to the log while plotting.
+    """
+    errmsg=[]
+
+    path, ext = os.path.splitext(filename)
+    verExts = [".png", ".pdf", ".svg", ".eps"]
+    if ext.lower() not in verExts:
```

Comment:
I would not add this here. We should not be responsible to decide which extensions `matplotlib` can accept. I suggest wrapping the `plt.savefig` call in a `try:catch` block and handling the error in case it crashes; for example, because of bad extension format.